src/flightControl/mainDraft.cpp

```diff
@@ -0,0 +1,22 @@
+#include "FSM1.0/FSM.h"
+
+class Control{
+private:
+    FSM fsm;
+public:
+    Control(){
+        fsm = FSM();
+    }
+    ~Control(){
+        fsm.~FSM(); 

```

Comment:
**THIS IS FORBIDDEN**

Reply:
File was obsolete and has no place in the code, deleted